util/MsSqlMigratorUtility/Program.cs

```diff
@@ -15,21 +15,23 @@ private static int Main(string[] args)
         [Option('r', "repeatable", Description = "Mark scripts as repeatable")]
         bool repeatable = false,
         [Option('f', "folder", Description = "Folder name of database scripts")]
-        string folderName = MigratorConstants.DefaultMigrationsFolderName)
-        => MigrateDatabase(databaseConnectionString, repeatable, folderName);
+        string folderName = MigratorConstants.DefaultMigrationsFolderName,
+        [Option("dry-run", Description = "Print the scripts that will be applied without actually executing them")]
```

Comment:
💭 We use shorthand for the others ... would `s` work? I see in the APT docs at least that "s" means "simulate" and then has an alias `dry-run`. Just trying to be consistent here. We can call it `dryRun` everywhere lower down.

Reply:
Maybe the `d` alias will suite here beter?